tests/depth_dependent_box_function_time_dependent.prm

```diff
@@ -0,0 +1,175 @@
+# Test depth-dependent material model using simple material model as
+# base model with depth-dependence specified by user-provided function
+
+# A description of convection in a 2d box. See the manual for more information.
```

Comment:
It looks like this is copied from somewhere. Can you either remove it, or just say which test/benchmark you copied?